src/Cpp/1-getting-started/1-HelloWindow/Application.cpp

```diff
@@ -0,0 +1,94 @@
+#include "Application.hpp"
+#include "Input/Input.hpp"
+
+#include <GLFW/glfw3.h>
+
+Application::Application(const std::string_view title)
+    : _input(nullptr), _window(nullptr), _width(0), _height(0), _title(title)
```

Comment:
I personally prefer direct member initialisation over initialisation in constructors, it's a lot clearer in terms of "missed initializers" (as you get an overview of all the members) rather than having to check back and forth whether a member is initialized or not